iModelCore/ECDb/ECDb/BuiltInFuncs.h

```diff
@@ -17,6 +17,7 @@ BEGIN_BENTLEY_SQLITE_EC_NAMESPACE
 #define SQLFUNC_ClassName "ec_classname"
 #define SQLFUNC_ClassId "ec_classid"
 #define SQLFUNC_InstanceOf "ec_instanceof"
+#define SQLFUNC_XmlCAToJson "XmlCAToJson"
```

Comment:
Affan suggests we rename all the ToSomtehing methods into underscore ones and keep aliases for the pre existing ones so it doesn't break. New name would be like "guid_str_to_binary"